libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -1414,6 +1420,9 @@ TEST_F(CppInterface, HlslFunctionality1SurvivesCloning) {
   CompileOptions options;
   options.SetSourceLanguage(shaderc_source_language_hlsl);
   options.SetHlslFunctionality1(true);
+  // The counter needs a binding, and there is no way to set it in the shader
```

Comment:
Is there a glslang bug associated we could link so we can remove this when vk::counter_binding is supported?

Reply:
I just made one: https://github.com/KhronosGroup/glslang/issues/1616

Reply:
Added comment to point to the Glslang enhancement request